hmri_calc_R2s.m

```diff
@@ -156,6 +156,12 @@
         [R2s0,A0]=hmri_calc_R2s(weighted_data,initmethod);
         
         if ~exist('opt','var')
+            %lsqcurvefit uses toolbox check for it:
```

Comment:
I think this check should be moved to the top of the case (i.e. line 150) so that it aborts before any calculation is done and only has to appear once.